Jenkinsfile

```diff
@@ -37,20 +37,33 @@ builders = pipeline_builder.createBuilders { container ->
     container.copyTo(pipeline_builder.project, pipeline_builder.project)
   }  // stage
 
+  pipeline_builder.stage("${container.key}: Conda") {
+    container.sh """
+      curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh --output miniconda.sh
```

Comment:
Should we have conda in the docker image instead of downloading every time?

Reply:
I was thinking the same thing as I will need this for just-bin-it and NICOS at some point.
@amues how does this sound to you?

Reply:
This sounds like a good idea. I'll add it to the image.

Reply:
Miniconda is now available at _/opt/miniconda_ in the version 5.1.0 of the image. The `update` step is run in the image build, but `conda init bash` is not.

Reply:
Thanks, I'll update the jenkinsfile to try it out

Reply:
I had forgotten to update the Jenkins shared library to use the new version of the container. I have just done that and will trigger a new build.